src/patches/parts/hotkeys.cc

```diff
@@ -515,7 +502,7 @@ void ChatMessageListLocalViewController_AboutToShow_Hook(ChatMessageListLocalVie
 
 void InitializeActions_Hook(auto original, void* _this)
 {
-  if (Config::Get().use_scopely_hotkeys && Config::Get().hotkeys_enabled) {
```

Comment:
What is the reasoning for removing this?
This is intended to disable hotkeys entirely, both the regular in-game ones plus ours.

Reply:
Someone mentioned something in the chat, not entirely sure what is was now..... I can restore it as I don't think it makes a difference to what I've done.